common/app/routes/challenges/components/step/Step.jsx

```diff
@@ -178,6 +190,7 @@ export class StepChallenge extends PureComponent {
         key={ imgUrl }
         >
         <a
+          className='fixed-height'
```

Comment:
Agreed. Note that almost all images are 1280x720, but we shouldn't force them in case they happen to be shallower.